bcp47/bcp47.cabal

```diff
@@ -86,7 +86,6 @@ test-suite doctest
   build-depends:
       base >=4.7 && <5
     , doctest
-    , megaparsec
```

Comment:
Huh, how about that.

Reply:
I was surprised by this, but I guess that's good?